demos/ros_examples/README.md

```diff
@@ -0,0 +1,34 @@
+# `ros_examples` demonstration scripts
+
+## Table of contents:
+* [Running a demo script](#running-demonstration-scripts)
+* [Task Space control loop example](#task_space_control_loop)
+
+## Running demonstration scripts
+This package contains the same control loop examples as in [`control_loop_examples`](../control_loop_examples) demos
+folder, but adapted for usage in a ROS environment.
+The whole folder is a fully functional ROS package that can be directly copied in a ROS workspace.
+There is also a provided `Dockerfile` that encapsulates the whole in a containerized ROS environment.
+Running the `run-demo.sh` script opens an interactive container with the ROS environment already setup for running
+the demonstration scripts.
+
+In order to run the demonstrations, a `roscore` needs to be running.
+One can be started from the interactive container.
+Running the scripts uses ROS commands, e.g. to run the `task_space_control_loop` script:
+
+```console
+rosrun ros_examples task_space_control_loop
+```
+
+You can also directly use the launchfile that start every requirement for the demo to run correctly:
```

Comment:
starts all the requirements? not sure about all the/every but certainly start with an s

Reply:
https://english.stackexchange.com/questions/75654/any-requirement-or-any-requirements so for requirement I guess singular is ok. start is indeed missing the s